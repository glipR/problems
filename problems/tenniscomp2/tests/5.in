256
91o35o5miys9ycyfza7z
wbw09gpo
xlmi29o2d6xq
xr5qclg9y2zw
mweykkjjk
csz863bursirhgkk
9sm9nn
asxfg5l2i4fp0jl5e
q8w7yz9owzu3urg
cjk0gjug1t
w0lyvtep3e5b6kqlt5
917u9
4h3zneyj5cpts2
cqun9gr3r427mg05
g3yx1dx9og
2zuq84l0rnjjj0kioo
nuzkwc688bwzmcdj4
mot6us7fuc
zoox3k4ibz8w
grhnaokj8877
i5xfxacuw435a
7o0kl3c3u9l6mebw8q2
n2bc7ztnb63pmrf54o
ie84p3za6ezu
v7sshx7yvkzrfg3fwq
qudkck05z0hdn
09ffzgp0qizrb7
lyshpvxkdxvp
vlg87edwleze8
xll0c4tbi
ri99pfquv8m
x2sbqhi9dh6q7
nvajri3dwg2bcux6yk
oza27ri89r274
aifoka7can
rer87f5yenxz8pjd0
rc18lpb9jyema
400yzlummsdqjebsf7q6
665otfowjoz3
kdjej4plb1bq37h
h7ae6t31lahojy
12ybde189riwhh11tq
gh8z3j4j35lq
zb3wcflkcc7fwymx69
1g0p3o103drrv8j
emc2qs
b6wq6kftohyisa1tyy
cbvti4kndz8k9wl9
txsxpmx2xd49jr
wy2mfklp6lucp
d9p27d0vhxtc
ms5yt
a8hnzkkyc
66fv861vgv5zy5x16y
lw11ks
zccupcyc6ccs6u
1bn1mz52duv
kj8gr9qd97q8e6f3x9df
dk8rhofe0w
v0jmeigtm8hk
86kjza72hzd69otsz5
v2sbon8debf88fabyceg
p1u2ji9aa7ttk
jh4gvq81jl7v
oj3asgpyhbsv4yc
7aek217uuh4azf
9kgn4obr1lu2o52
amwu53shc
k2nx4
t2cbq3l5rs53d2y8
prtrs3v
0rbzi
puk3yho0d9pgd36kx
r9xfu0q87
i0sfss4wlejhi
d8ioupd40ef
u3hpb2wztef
d4cggg99vxphll3tw3
1jcamt7hk
rs80zltyzq9lwnmgev
6d6dphes
t37sjg3h4e0hpc
4rtf7w996y2to8zjfj
8oxnd
rdv1cea1iuw5qx5qltr
rv50yzt4wzx
qn1tlpl59vh737zwhl
qs2obi1ius3lqgt6xuk
yf8dk
cd0h3arqm
gt4wdiltnwxmnc89ew
s7sjbcya
lvb8w45uq76gwwll5
u1r61crtongo
inoo6s2xhz32becx
yd728g0ok
lmjtmw9mpznuge3mdf
zssalj47
210wl7yjub
1rdy12g3bb7bd6d33
cd9fy
sneqo5xywm9vb671rk2n
uhmf2m4juh6jxl404mm9
4vnv95np9ek1g3xxftyo
lmphc93ct6l3qs
5piow3j
swzkx5a181pg
vk2k58qanlni
m9k4iop3o2ykvl4fnfj
vp6qnnbaqqxnuwmu
dcwgsfroivj0he5u2
xuaijarz
o1en71kocxlqe6
idwjrv0o7k03
wjei66t1xh54
kvmybh29m5hi2ozfx70
8wsmpx
z3xlug297pnz3p9
ubjvrex91
dpnh8gsbjkzbvz
dfsut9qmlgv07pgvq
wmmc5v8ay7k2yh
6fwrbuiotiqwe4
vstin1fttrh1hm6d0
t43wveyhjobiffobu
tgou59o4uypvkcy7vaj1
dhtl7b3v
zjboef8p
m11kuw703ryz6t
56le3fn
7v29i4a29va9
c3qcwz72qdnfd
rjk8jh
epoe8sc53qa
6xts879edj1t328y
v25fsnxqgm6h2w33q2u3
yfv44vw
mqp0678m3
igptip1s6
porxpnrcupv
67h6eav2
2ox5xu
v0gas9q20hk9b0rd3j
meghjfnpu8h4yic7a
47ldz
wwzwal44g2r4k2lku
ag91h859pkhc2
niueksa05
rxezesrzw66s
xg0n4y9
rmdaa41y00le7rzvia
7e9m35xgq6tbpviq45
e9vx6kd19ryiqjkvjth
a3banxky
73sliocqc7k1q87xe
07jvwgelhe
he9d1bspdo9gzhmadye
ffyon184kze4
swukirr
fmgphb92g88m9xz
bcsm4p
rmt1m5w5nxe
9fuznn2e2mmb1
5j0i25b95kv16kr
sjrb2ywn
wdumq7fxr9vhhzz
wrm38nn72y67
2vvimv07hyukhzwe
2qff4rm
2fp6h2yha5mak4v1z
gzw6695motkg0
nriec4qb08stvdur
6z9w705jliifpw04r7
z44vk6e
3q2tx7enksnr
2oa2s9o9h6
9smmfp620qg
4e0po4hyg1arf3fosu
v7t4sad3c13ru3lt6db6
49vox4g7l0mgwhle4sxc
o44dc5chj
hmiix82o
u4e7jtn5c0gxf862
e086uybpyyplg86o8x
j7pyf9xxsfqde737h
ebm2fjhwy0fgmltlsek
jpfwh3iy8znfy
pziklqr4u
wxduq08wy3zdkfp6n6w
id88srpg9yy00261k
ond64j8vn
of8wgwbep
4ut1zgc21pbqfwgh16q
brq9iaiu2cmmcg
kjjcgn639eyr40c9qlq0
5d2i6g
a6idpky7kjhnv
uj0h864omgv1
84e9xop4al0wdv2x8
rppv10j5umia
h6jafitm9o
vylmnullwojx5o
qiulfoaxpjwlplm
6nvr30i3pb
xlwh6mu9sgbpgz3
1gkej5xp67bajy0xr
inpiba
rdpq0obbgiq17q
tmrtbj
lle22z0je59
60p9id3d8frr
2euabrff5dym9wwrg1
ccf5523v4umzsf
zsgy3696xltij50590o
vcxlo1umo7n0bd3kjs
so6mvr8h3xnfu1nws2
j07f46y
fio92vpzv
nwvxuvin48blecfi4nf
oexgk8
01vxe6s9slypm
tvsh0ziq0jqlhxxk7c
i2fnxaf746gcpc
3jvuwlp1
gr07f3qbs
kidj9isrib66zcjf
i20blx1ixgdz
v2ksppfj6gzdaqb15ib
4m8h1qs6gt0ffdex
f3tkj5rpi00npxj
8ngvoquswpzwuya
5ionu9e0te3w0wokq0d
tnnzip2l1
l7cwsyge82gggjx94
38fme3f6jfl
ix1lz1id
njj53nb
lo09njc2ak12nyod
zk9uyg
sjvd0thh70ai3eejyc
cc385u4awx1n3h
1pejuabut
6rt50
74caqyinhwtz7uzs6gm
gnv2ty
f36unpwfzk
qqb3gvk45
g80nf2o57fz2tk8tc
h16uhd7017dwh6
4z2813g
lwyjpr139r0o68
01kex5qq4yxxlukbs
gx0dcryqz
o3ohprv4ve0w6yw54n1
p3gumtxanjlc
rfeg9zasjikyz4a2run9
zsgy3696xltij50590o defeats ccf5523v4umzsf
u4e7jtn5c0gxf862 defeats e086uybpyyplg86o8x
j7pyf9xxsfqde737h defeats ebm2fjhwy0fgmltlsek
210wl7yjub defeats 1rdy12g3bb7bd6d33
84e9xop4al0wdv2x8 defeats a6idpky7kjhnv
nriec4qb08stvdur defeats gzw6695motkg0
porxpnrcupv defeats igptip1s6
gnv2ty defeats g80nf2o57fz2tk8tc
p3gumtxanjlc defeats o3ohprv4ve0w6yw54n1
cjk0gjug1t defeats q8w7yz9owzu3urg
ri99pfquv8m defeats x2sbqhi9dh6q7
lvb8w45uq76gwwll5 defeats u1r61crtongo
9sm9nn defeats mweykkjjk
zjboef8p defeats idwjrv0o7k03
4e0po4hyg1arf3fosu defeats 9smmfp620qg
ix1lz1id defeats 38fme3f6jfl
dpnh8gsbjkzbvz defeats ubjvrex91
xlmi29o2d6xq defeats wbw09gpo
6fwrbuiotiqwe4 defeats vstin1fttrh1hm6d0
z44vk6e defeats 6z9w705jliifpw04r7
o3ohprv4ve0w6yw54n1 defeats gx0dcryqz
wy2mfklp6lucp defeats txsxpmx2xd49jr
jh4gvq81jl7v defeats p1u2ji9aa7ttk
idwjrv0o7k03 defeats kvmybh29m5hi2ozfx70
cqun9gr3r427mg05 defeats cjk0gjug1t
1pejuabut defeats cc385u4awx1n3h
i0sfss4wlejhi defeats zjboef8p
4rtf7w996y2to8zjfj defeats qs2obi1ius3lqgt6xuk
jh4gvq81jl7v defeats v2sbon8debf88fabyceg
2zuq84l0rnjjj0kioo defeats g3yx1dx9og
2ox5xu defeats 67h6eav2
oexgk8 defeats fio92vpzv
7aek217uuh4azf defeats 9kgn4obr1lu2o52
wxduq08wy3zdkfp6n6w defeats id88srpg9yy00261k
d4cggg99vxphll3tw3 defeats 1jcamt7hk
inoo6s2xhz32becx defeats yd728g0ok
zjboef8p defeats dhtl7b3v
a3banxky defeats e9vx6kd19ryiqjkvjth
i0sfss4wlejhi defeats d4cggg99vxphll3tw3
kidj9isrib66zcjf defeats 4m8h1qs6gt0ffdex
9sm9nn defeats cqun9gr3r427mg05
v2sbon8debf88fabyceg defeats 86kjza72hzd69otsz5
7e9m35xgq6tbpviq45 defeats rmdaa41y00le7rzvia
4rtf7w996y2to8zjfj defeats 8oxnd
xlwh6mu9sgbpgz3 defeats lle22z0je59
gh8z3j4j35lq defeats zb3wcflkcc7fwymx69
5d2i6g defeats 4ut1zgc21pbqfwgh16q
porxpnrcupv defeats meghjfnpu8h4yic7a
mweykkjjk defeats csz863bursirhgkk
5ionu9e0te3w0wokq0d defeats 8ngvoquswpzwuya
nuzkwc688bwzmcdj4 defeats ri99pfquv8m
07jvwgelhe defeats 73sliocqc7k1q87xe
sjvd0thh70ai3eejyc defeats ix1lz1id
porxpnrcupv defeats m11kuw703ryz6t
lmphc93ct6l3qs defeats 210wl7yjub
of8wgwbep defeats ond64j8vn
7v29i4a29va9 defeats c3qcwz72qdnfd
wrm38nn72y67 defeats 2vvimv07hyukhzwe
210wl7yjub defeats zssalj47
ix1lz1id defeats tnnzip2l1
v0jmeigtm8hk defeats dk8rhofe0w
6xts879edj1t328y defeats v25fsnxqgm6h2w33q2u3
xll0c4tbi defeats vlg87edwleze8
3q2tx7enksnr defeats porxpnrcupv
g80nf2o57fz2tk8tc defeats qqb3gvk45
xlwh6mu9sgbpgz3 defeats rdpq0obbgiq17q
wy2mfklp6lucp defeats lw11ks
sjvd0thh70ai3eejyc defeats zk9uyg
rdpq0obbgiq17q defeats inpiba
h7ae6t31lahojy defeats gh8z3j4j35lq
lmphc93ct6l3qs defeats m9k4iop3o2ykvl4fnfj
9sm9nn defeats asxfg5l2i4fp0jl5e
h7ae6t31lahojy defeats 12ybde189riwhh11tq
s7sjbcya defeats gt4wdiltnwxmnc89ew
60p9id3d8frr defeats 2euabrff5dym9wwrg1
wbw09gpo defeats 91o35o5miys9ycyfza7z
xlwh6mu9sgbpgz3 defeats 5d2i6g
zoox3k4ibz8w defeats grhnaokj8877
m9k4iop3o2ykvl4fnfj defeats xuaijarz
swzkx5a181pg defeats vk2k58qanlni
cd0h3arqm defeats s7sjbcya
4z2813g defeats p3gumtxanjlc
01vxe6s9slypm defeats tvsh0ziq0jqlhxxk7c
sjrb2ywn defeats wrm38nn72y67
qiulfoaxpjwlplm defeats 6nvr30i3pb
rc18lpb9jyema defeats kdjej4plb1bq37h
3q2tx7enksnr defeats 2fp6h2yha5mak4v1z
so6mvr8h3xnfu1nws2 defeats vcxlo1umo7n0bd3kjs
ri99pfquv8m defeats qudkck05z0hdn
4e0po4hyg1arf3fosu defeats u4e7jtn5c0gxf862
6rt50 defeats gnv2ty
7aek217uuh4azf defeats oj3asgpyhbsv4yc
sjrb2ywn defeats 9fuznn2e2mmb1
2fp6h2yha5mak4v1z defeats 2qff4rm
6rt50 defeats kidj9isrib66zcjf
cd0h3arqm defeats lvb8w45uq76gwwll5
nuzkwc688bwzmcdj4 defeats mot6us7fuc
a6idpky7kjhnv defeats uj0h864omgv1
2fp6h2yha5mak4v1z defeats nriec4qb08stvdur
wy2mfklp6lucp defeats ms5yt
ag91h859pkhc2 defeats niueksa05
210wl7yjub defeats uhmf2m4juh6jxl404mm9
a3banxky defeats 07jvwgelhe
lmphc93ct6l3qs defeats swzkx5a181pg
9kgn4obr1lu2o52 defeats amwu53shc
xlmi29o2d6xq defeats xr5qclg9y2zw
5d2i6g defeats kjjcgn639eyr40c9qlq0
0rbzi defeats k2nx4
lle22z0je59 defeats oexgk8
4z2813g defeats 01kex5qq4yxxlukbs
cd0h3arqm defeats yf8dk
z3xlug297pnz3p9 defeats 8wsmpx
6rt50 defeats xlwh6mu9sgbpgz3
cqun9gr3r427mg05 defeats 4h3zneyj5cpts2
cqun9gr3r427mg05 defeats 2zuq84l0rnjjj0kioo
wwzwal44g2r4k2lku defeats xg0n4y9
nuzkwc688bwzmcdj4 defeats 9sm9nn
he9d1bspdo9gzhmadye defeats ffyon184kze4
3jvuwlp1 defeats i2fnxaf746gcpc
kvmybh29m5hi2ozfx70 defeats wjei66t1xh54
wwzwal44g2r4k2lku defeats 47ldz
oexgk8 defeats nwvxuvin48blecfi4nf
60p9id3d8frr defeats lle22z0je59
i0sfss4wlejhi defeats d8ioupd40ef
cd9fy defeats sneqo5xywm9vb671rk2n
wxduq08wy3zdkfp6n6w defeats of8wgwbep
i0sfss4wlejhi defeats nuzkwc688bwzmcdj4
4rtf7w996y2to8zjfj defeats t37sjg3h4e0hpc
kidj9isrib66zcjf defeats sjvd0thh70ai3eejyc
porxpnrcupv defeats wwzwal44g2r4k2lku
6rt50 defeats 1pejuabut
ms5yt defeats d9p27d0vhxtc
sjvd0thh70ai3eejyc defeats lo09njc2ak12nyod
v0jmeigtm8hk defeats 1bn1mz52duv
lw11ks defeats a8hnzkkyc
gnv2ty defeats f36unpwfzk
5d2i6g defeats 84e9xop4al0wdv2x8
oza27ri89r274 defeats nvajri3dwg2bcux6yk
idwjrv0o7k03 defeats z3xlug297pnz3p9
wxduq08wy3zdkfp6n6w defeats 4e0po4hyg1arf3fosu
84e9xop4al0wdv2x8 defeats rppv10j5umia
nuzkwc688bwzmcdj4 defeats zoox3k4ibz8w
0rbzi defeats prtrs3v
ie84p3za6ezu defeats n2bc7ztnb63pmrf54o
zjboef8p defeats 6fwrbuiotiqwe4
rv50yzt4wzx defeats rdv1cea1iuw5qx5qltr
kdjej4plb1bq37h defeats 665otfowjoz3
6rt50 defeats 4z2813g
lmphc93ct6l3qs defeats 5piow3j
rer87f5yenxz8pjd0 defeats aifoka7can
porxpnrcupv defeats mqp0678m3
1bn1mz52duv defeats kj8gr9qd97q8e6f3x9df
4z2813g defeats h16uhd7017dwh6
3jvuwlp1 defeats 01vxe6s9slypm
nuzkwc688bwzmcdj4 defeats ie84p3za6ezu
ri99pfquv8m defeats xll0c4tbi
i20blx1ixgdz defeats v2ksppfj6gzdaqb15ib
zsgy3696xltij50590o defeats so6mvr8h3xnfu1nws2
p3gumtxanjlc defeats rfeg9zasjikyz4a2run9
uhmf2m4juh6jxl404mm9 defeats 4vnv95np9ek1g3xxftyo
m9k4iop3o2ykvl4fnfj defeats vp6qnnbaqqxnuwmu
6rt50 defeats 74caqyinhwtz7uzs6gm
emc2qs defeats b6wq6kftohyisa1tyy
4e0po4hyg1arf3fosu defeats v7t4sad3c13ru3lt6db6
zjboef8p defeats lmphc93ct6l3qs
qiulfoaxpjwlplm defeats h6jafitm9o
h7ae6t31lahojy defeats emc2qs
t43wveyhjobiffobu defeats tgou59o4uypvkcy7vaj1
k2nx4 defeats t2cbq3l5rs53d2y8
wy2mfklp6lucp defeats rer87f5yenxz8pjd0
qudkck05z0hdn defeats v7sshx7yvkzrfg3fwq
4m8h1qs6gt0ffdex defeats f3tkj5rpi00npxj
hmiix82o defeats o44dc5chj
9sm9nn defeats xlmi29o2d6xq
wy2mfklp6lucp defeats jh4gvq81jl7v
9fuznn2e2mmb1 defeats 5j0i25b95kv16kr
7aek217uuh4azf defeats 0rbzi
6fwrbuiotiqwe4 defeats dfsut9qmlgv07pgvq
i0sfss4wlejhi defeats cd0h3arqm
3q2tx7enksnr defeats 2oa2s9o9h6
xg0n4y9 defeats 7e9m35xgq6tbpviq45
i0sfss4wlejhi defeats 7aek217uuh4azf
xuaijarz defeats dcwgsfroivj0he5u2
xg0n4y9 defeats rxezesrzw66s
rer87f5yenxz8pjd0 defeats rc18lpb9jyema
9fuznn2e2mmb1 defeats bcsm4p
m11kuw703ryz6t defeats 7v29i4a29va9
b6wq6kftohyisa1tyy defeats cbvti4kndz8k9wl9
wwzwal44g2r4k2lku defeats ag91h859pkhc2
kidj9isrib66zcjf defeats gr07f3qbs
rer87f5yenxz8pjd0 defeats h7ae6t31lahojy
jh4gvq81jl7v defeats v0jmeigtm8hk
lle22z0je59 defeats tmrtbj
m11kuw703ryz6t defeats 56le3fn
xlwh6mu9sgbpgz3 defeats 1gkej5xp67bajy0xr
fmgphb92g88m9xz defeats swukirr
u4e7jtn5c0gxf862 defeats hmiix82o
meghjfnpu8h4yic7a defeats 2ox5xu
v7t4sad3c13ru3lt6db6 defeats 49vox4g7l0mgwhle4sxc
3q2tx7enksnr defeats wxduq08wy3zdkfp6n6w
rjk8jh defeats epoe8sc53qa
qs2obi1ius3lqgt6xuk defeats rv50yzt4wzx
nuzkwc688bwzmcdj4 defeats wy2mfklp6lucp
3q2tx7enksnr defeats z44vk6e
lw11ks defeats zccupcyc6ccs6u
jpfwh3iy8znfy defeats pziklqr4u
1jcamt7hk defeats rs80zltyzq9lwnmgev
lle22z0je59 defeats zsgy3696xltij50590o
mqp0678m3 defeats yfv44vw
t37sjg3h4e0hpc defeats 6d6dphes
uhmf2m4juh6jxl404mm9 defeats cd9fy
lvb8w45uq76gwwll5 defeats inoo6s2xhz32becx
w0lyvtep3e5b6kqlt5 defeats 917u9
ie84p3za6ezu defeats i5xfxacuw435a
z3xlug297pnz3p9 defeats dpnh8gsbjkzbvz
h6jafitm9o defeats vylmnullwojx5o
wwzwal44g2r4k2lku defeats fmgphb92g88m9xz
i0sfss4wlejhi defeats puk3yho0d9pgd36kx
d4cggg99vxphll3tw3 defeats u3hpb2wztef
6rt50 defeats i0sfss4wlejhi
zjboef8p defeats t43wveyhjobiffobu
tnnzip2l1 defeats l7cwsyge82gggjx94
wxduq08wy3zdkfp6n6w defeats jpfwh3iy8znfy
09ffzgp0qizrb7 defeats lyshpvxkdxvp
a8hnzkkyc defeats 66fv861vgv5zy5x16y
01kex5qq4yxxlukbs defeats lwyjpr139r0o68
jpfwh3iy8znfy defeats j7pyf9xxsfqde737h
rer87f5yenxz8pjd0 defeats oza27ri89r274
xlwh6mu9sgbpgz3 defeats qiulfoaxpjwlplm
4m8h1qs6gt0ffdex defeats 5ionu9e0te3w0wokq0d
qudkck05z0hdn defeats 09ffzgp0qizrb7
kidj9isrib66zcjf defeats i20blx1ixgdz
3q2tx7enksnr defeats sjrb2ywn
fmgphb92g88m9xz defeats he9d1bspdo9gzhmadye
fio92vpzv defeats j07f46y
i5xfxacuw435a defeats 7o0kl3c3u9l6mebw8q2
zssalj47 defeats lmjtmw9mpznuge3mdf
dfsut9qmlgv07pgvq defeats wmmc5v8ay7k2yh
oexgk8 defeats 3jvuwlp1
m11kuw703ryz6t defeats 6xts879edj1t328y
puk3yho0d9pgd36kx defeats r9xfu0q87
cjk0gjug1t defeats w0lyvtep3e5b6kqlt5
idwjrv0o7k03 defeats o1en71kocxlqe6
6xts879edj1t328y defeats rjk8jh
cd0h3arqm defeats 4rtf7w996y2to8zjfj
qs2obi1ius3lqgt6xuk defeats qn1tlpl59vh737zwhl
bcsm4p defeats rmt1m5w5nxe
lo09njc2ak12nyod defeats njj53nb
rc18lpb9jyema defeats 400yzlummsdqjebsf7q6
4ut1zgc21pbqfwgh16q defeats brq9iaiu2cmmcg
emc2qs defeats 1g0p3o103drrv8j
fmgphb92g88m9xz defeats a3banxky
meghjfnpu8h4yic7a defeats v0gas9q20hk9b0rd3j
6rt50 defeats 3q2tx7enksnr
sjrb2ywn defeats wdumq7fxr9vhhzz
