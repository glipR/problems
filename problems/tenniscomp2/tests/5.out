60p9id3d8frr must be the cheater
They cheated on 1 games
60p9id3d8frr defeats lle22z0je59
